wsrep_api.h

```diff
@@ -246,6 +258,19 @@ extern int
 wsrep_gtid_print(const wsrep_gtid_t* gtid, char* str, size_t str_len);
 
 /*!
+ * @brief Compare two GTIDs
+ *
+ * First the UUID components are compared according to wsrep_uuid_compare().
+ * If they match, the return value is -1, 0, 1 if the SEQNO component of lhs
+ * is respectively smaller, equal, or greater than the SEQNO component of rhs.
+ *
+ * @return -1, 0, 1 if lhs is respectively smaller, equal, or greater than rhs
+ */
+extern int
+wsrep_gtid_compare (const wsrep_gtid_t* lhs, const wsrep_gtid_t* rhs);
```

Comment:
The reason we don't have this in gu:: is a poor use semantics in that case: what if UUIDs differ? In a program you probably don't even want to reach this point before you compare UUIDs separately and make sure they are equal, So at this point all is left is to compare seqnos.